app/views/incomeTaxHistory/IncomeTaxHistoryView.scala.html

```diff
@@ -45,22 +46,40 @@
     </p>
 }
 
+
+@printHeader = @{
+    printBanner(preHeadingText = request.fullName)
+}
+
+
+
 @main(
     title = messages("tai.incomeTax.history.pageTitle"),
     authedUser = Some(request.taiUser),
     pageTitle = Some(Messages("tai.service.navTitle")),
     backLinkUrl = Some(routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage.url),
     backLinkContent = Some(Messages("your.paye.income.tax.overview")),
-    pagePrintable = true,
-    pagePrintName = Some("historic-pay-as-you-earn"),
+    pagePrintable = true
 ){
+
+  @printHeader
```

Comment:
is it worth adding a test to the ViewSpec for this?

Reply:
we can't test for the print banner elements as they are hidden in the browser and only become visible when the media changes to print -  we can probably add a test for the print button though